### ISSUE TITLE
Memory Error when loading training arrays into memory

### ISSUE THREAD

**Issue by gabrieltseng** [esowc/ml_drought#38]
Now that we are including the GLEAM dataset in our training data, the data is too large to fit into memory.

The solution is probably a dataloader, but this makes normalization more tricky.
